src/Utility/Language.php

```diff
@@ -9,37 +9,75 @@ class Language {
 
 	/**
 	 * @param string|null $languageList List of language codes or locales codes.
-	 * @param bool $forceLowerCase Flag to auto lower case country part of locale codes.
+	 * @param array|bool|null $options Flags to forceAllLowerCase or keepDuplicates.
+	 *        @deprecated: Set to true/false to toggle forceAllLowerCase
+	 * @param bool $keepDuplicates Flag to keep or discard duplicates, defaults to keep.
 	 *
 	 * @return array
 	 */
-	public static function parseLanguageList($languageList = null, $forceLowerCase = true) {
+	public static function parseLanguageList($languageList = null, Array $options = []) {
+		$defaultOptions = [
+			'forceAllLowerCase' => true,
+			'keepDuplicates' => true,
```

Comment:
because of BC I had forceAllLowerCase set to true by default... 
I can change it for removeDuplicates no problem